packages/context/src/lib/decorators/consume.ts

```diff
@@ -47,18 +47,19 @@ export function consume<ValueType>({
   context: Context<unknown, ValueType>;
   subscribe?: boolean;
 }): ConsumeDecorator<ValueType> {
-  return (<C extends ReactiveElement, V extends ValueType>(
-    protoOrTarget: ClassAccessorDecoratorTarget<C, V>,
-    nameOrContext: PropertyKey | ClassAccessorDecoratorContext<C, V>
+  return ((
+    protoOrTarget: ClassAccessorDecoratorTarget<ReactiveElement, ValueType>,
+    nameOrContext:
+      | PropertyKey
+      | ClassAccessorDecoratorContext<ReactiveElement, ValueType>
```

Comment:
Nice. I guess the generic wasn't necessary since we're casting the type of the returned function anyway. Perhaps we want to do the same change to the `@provide()` decorator just for consistency's sake?

Reply:
Could you push the change? I don't have time too, apologies!

Reply:
Sure thing.